### ISSUE TITLE
Create unit tests for Resources component

### ISSUE THREAD

**Issue by sunithapatel** [frontendstudygroup/frontendstudygroup.github.io#48]
Create unit tests for the Resources component. 

Feel free to list out test scenarios here first for feedback if you like.

**Comment by IngridFuentes** [comment]
Hey @sunithapatel I want to learn how to create unit tests!! Can I work on this with some guidance? Like if you know a good documentation to start. Thanks!

**Comment by princiya** [comment]
@IngridFuentes good one.
we are using react-testing-library. 
https://testing-library.com/docs/react-testing-library/intro/ So this is a good place to get started. Feel free to ask more questions.

**Comment by Priyami** [comment]
- Started to write test cases for the Resources component. 
- Need some feedback.  


<img width="1290" alt="TestCases" src="https://user-images.githubusercontent.com/4775877/137564188-27de9045-5a6d-4ddb-a9b7-bf5102b413d5.png">

**Comment by princiya** [comment]
@Priyami how did you arrive at the above table?
are you doing some manual testing too?

Also, can you and @IngridFuentes sync on this issue. Since @IngridFuentes too has claimed the issue, I believe she too has some ideas or doing some work. It's better for both of you to talk & coordinate before create PRs to avoid any duplicate work.

**Comment by IngridFuentes** [comment]
Yes, I have a few test cases written, so if you want to show me what you have, so we don't do the same @Priyami :)
